axelrod/tests/unit/test_moran.py

```diff
@@ -151,30 +261,15 @@ def test_three_players_with_mutation(self):
     def test_four_players(self):
         players = [axelrod.Cooperator() for _ in range(3)]
         players.append(axelrod.Defector())
-        random.seed(10)
+        axelrod.seed(29)
         mp = MoranProcess(players)
         populations = mp.play()
         self.assertEqual(len(mp), 9)
         self.assertEqual(len(populations), 9)
         self.assertEqual(populations, mp.populations)
         self.assertEqual(mp.winning_strategy_name, str(axelrod.Defector()))
 
-    def test_standard_fixation(self):
```

Comment:
Actually, I don't understand what that test was doing at all with the Mock class (apart from testing that it worked). I'm fixed the py3.6 problems and about to push with a version of this test (just using a normal match).